package.json

```diff
@@ -34,11 +34,12 @@
     "@typescript-eslint/parser": "^5.62.0",
     "@vercel/ncc": "^0.36.1",
     "eslint": "^8.45.0",
-    "eslint-plugin-github": "^4.8.0",
+    "eslint-plugin-github": "^4.9.2",
     "eslint-plugin-jest": "^27.2.3",
+    "eslint-plugin-prettier": "^5.0.0",
```

Comment:
This dependency is now required (probably because the `eslint-plugin-github` config includes `prettier/prettier`).